terraform/dev/main.tf

```diff
@@ -50,8 +50,8 @@ data "cloudfoundry_service" "rds" {
 resource "cloudfoundry_service_instance" "database" {
   name             = "tdp-db-dev"
   space            = data.cloudfoundry_space.space.id
-  service_plan     = data.cloudfoundry_service.rds.service_plans["micro-psql"]
-  json_params      = "{\"version\": \"15\"}"
+  service_plan     = data.cloudfoundry_service.rds.service_plans["medium-gp-psql"]
+  json_params      = "{\"version\": \"15\", \"storage_type\": \"gp3\", \"storage\": 50}"
```

Comment:
Added TF updates strictly to allow Alex to keep testing reparsing which requires the above service changes to function.

Reply:
Will remove once we are ready to merge